mock_service/lib/dsl.py

```diff
@@ -12,12 +12,15 @@
 # ********************************************************************************/
 
 import logging
-from typing import Any, Callable, Dict, List
+from typing import Any, Callable, Dict, List, Optional
 
 from lib.action import Action, ActionContext, AnimationAction, SetAction
 from lib.animator import RepeatMode
 from lib.behavior import Behavior, ExecutionContext
-from lib.trigger import EventTriggerResult, Trigger
+from lib.trigger import EventTriggerResult, Trigger, EventType, EventTrigger
+
+# Set the log level to suppress log messages because we call connect/disconnect of client quite often
+logging.getLogger("kuksa_client").setLevel(logging.WARNING)
```

Comment:
A modification of a completely unrelated logger should definitely not be located in the public DSL module. Move it to mockservice.py?